Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Azure_Enrichment.yml

```diff
@@ -6,27 +6,21 @@ starttaskid: "0"
 tasks:
   "0":
     id: "0"
-    taskid: 2d64b816-89fc-41a2-854f-bb692f819049
+    taskid: cf72405b-002e-45ab-864b-93d9983d156a
```

Comment:
General questions: is it possible to use the hierarchy info to update the "Are there any matches from accepted risk lists?" task in the parent?  (bypass known Azure tenant/management group names). If so should be update it and/or how easy would it be?

Reply:
I think so in theory if I am understanding you correctly. I would like to understand the use case more just so I can give a clear answer.

Reply:
Updated in [f1689c2](https://github.com/demisto/content/pull/35384/commits/f1689c23d02bd02359dc48cf10fdf60d89e77b4b)